src/test/resources/features/refapp-2.x/03-trackPatient/EditPatientRelationship.feature

```diff
@@ -0,0 +1,13 @@
+ Feature: EditRelationshipType
+ 
+ Given System loads clinicianFacingPatientDashboardPage
+    When  User clicks on editRegistrationInformation
+    And   User clicks on EditRelationships
+    And  User click on find Relationships
+    Then system loads back   to patientDashboard
+
+    Examples:
```

Comment:
Are you sure you need to include examples

Reply:
Ofcourse we need them

Reply:
What would happen if you do not include example section in the feature file

Reply:
Feel free to checkout gherkin references https://cucumber.io/docs/gherkin/reference/ , as you have been looking to other examples written, we must write a feature file, and one main files to include is gherkin references these files are written in line with steps class, cc @k-joseph  might clarify more about this